apps/overige-objecten-api/src/utils/processData.test.ts

```diff
@@ -312,14 +312,17 @@ describe('processData', () => {
     const result = processData({ data });
     const outputHtml = result[0].notice;
     const dom = new JSDOM(outputHtml);
-    const sectionElement = dom.window.document.querySelector('section') as HTMLElement;
-    expect(sectionElement).not.toBeNull();
-    const h3Element = dom.window.document.querySelector('h2') as HTMLHeadingElement;
-    expect(h3Element).not.toBeNull();
-    expect(h3Element.textContent).toBe('FAQ Title');
-    const pElement = dom.window.document.querySelector('p') as HTMLParagraphElement;
-    expect(pElement).not.toBeNull();
-    expect(pElement.textContent).toBe('FAQ Inhoud');
+    const detailsElement = dom.window.document.querySelector('details') as HTMLElement;
+    const summaryElement = detailsElement.querySelector('summary') as HTMLElement;
+    const detailsContentElement = detailsElement.querySelector('div') as HTMLElement;
+    const paragraphElement = detailsContentElement.querySelector('p') as HTMLParagraphElement;
+    const heading = summaryElement.querySelector('h2') as HTMLHeadingElement;
+
+    expect(heading.textContent).toBe('FAQ Title');
```

Comment:
Dan moet je hier ook een extra check toevoegen
```suggestion
    expect(heading?.textContent).toBe('FAQ Title');
```